src/tactic/core.lean

```diff
@@ -498,17 +498,35 @@ meta def revert_deps (e : expr) : tactic ℕ := do
 meta def is_local_def (e : expr) : tactic unit :=
 retrieve $ do revert e, expr.elet _ _ _ _ ← target, skip
 
+private meta def partition_local_deps_aux (vs : list expr) : list expr → list expr → list (list expr)
```

Comment:
Is there any reason to make this `meta`?

Reply:
Yes, you can't use `expr` outside `meta`.

Reply:
There's actually nothing `expr`-specific about this function, it could be defined for any type with decidable equality. But I'm not sure what the generic use case is.

Reply:
None except that we manipulate `expr`. Anything with `decidable_eq` will do here however.